docker-assets/initialize-httpd-auth.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+HTTPD_AUTH_CONFIG_DIR=/etc/httpd/auth-conf.d
+AUTH_CONFIG_FILE="${HTTPD_AUTH_CONFIG_DIR}/auth-configuration.conf"
+SSSD_CONF_FILE="/etc/sssd/sssd.conf"
```

Comment:
If we are expecting specific file paths for particular authentication types, should we validate that the required files are present based on the type requested or is that too much to keep track of?